mesheryctl/internal/cli/root/system/start.go

```diff
@@ -79,174 +84,309 @@ func ValidateComposeFileForRecreation(CurrentServices map[string]utils.Service,
 	return nil
 }
 
+// applyManifest is a wrapper function for client.ApplyManifest
+func applyManifest(manifest []byte, client *meshkitkube.Client) error {
+	// ApplyManifest applies the given manifest file to the Kubernetes cluster
+	err := client.ApplyManifest(manifest, meshkitkube.ApplyOptions{
+		Namespace: "default",
+		Update:    true,
+		Delete:    false,
+	})
+
+	if err != nil {
+		return errors.Wrap(err, "failed to apply manifests")
+	}
+	return nil
+}
+
+// ApplyManifestFiles applies all the required manifests into the Kubernetes cluster
+func ApplyManifestFiles(requestedAdapters []string, client *meshkitkube.Client) error {
+	// path to the manifest files ~/.meshery/manifests
+	manifestFiles := filepath.Join(utils.MesheryFolder, utils.ManifestsFolder)
+
+	// read the manifest files as strings
+	// other than the adapters, meshery-service.yaml and service-account.yaml should be applied
+	mesheryServiceManifest, err := ioutil.ReadFile(filepath.Join(manifestFiles, utils.MesheryService))
```

Comment:
@navendu-pottekkat I wonder whether you have forgotten to deploy `manifests/meshery-deployment.yaml` ?

Reply:
yes 🤦‍♂️. I did not see such a file. Thanks for pointing it out!

Reply:
Is it added back @navendu-pottekkat